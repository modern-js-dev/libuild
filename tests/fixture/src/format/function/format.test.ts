import { getLibuilderTest } from '@modern-js/libuild-test-toolkit';

describe('fixture:format:function', () => {
  it('format function', async () => {
    const bundler = await getLibuilderTest({
      root: __dirname,
      format(chunkName: string) {
        if (chunkName.indexOf('index') > -1) {
          return 'cjs';
        }
        return 'esm';
      },
    });
    await bundler.build();

    bundler.expectJSOutputMatchSnapshot();
  });
});
