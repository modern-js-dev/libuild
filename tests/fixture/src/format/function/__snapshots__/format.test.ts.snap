// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`fixture:format:function format function 1`] = `
"// index.ts
function a() {
  console.log(1);
}
export {
  a
};
"
`;
